config/locales/en.yml

```diff
@@ -78,6 +78,7 @@ en:
       submit: Log in
       email: Email
       password: Password
+      remember_me: Remember me on this computer
```

Comment:
"on this computer" là sai nha, "on this browser" hoặc chuyển là "Remember me" thôi cũng được mà ta?